gateway/1.4/modules/ROOT/pages/flex-view-replica-status.adoc

```diff
@@ -5,21 +5,49 @@ endif::[]
 :page-aliases: runtime-manager::flex-gateway-replica-status.adoc
 :imagesdir: ../assets/images
 
-To view information about the status of Flex Replicas:
+In Runtime Manager, you can view the status of groups of Flex Replicas and individual Flex Replicas.
+
+== Groups of Flex Replicas
+
+To view information about the status of groups of Flex Replicas:
 
 . From Anypoint Platform, select *Runtime Manager*.
-. Select *Flex Gateway* in the navigation menu.
+. Select *Flex Gateway* in the navigation menu. +
+By default, all Flex Gateways regardless of their status are shown. To filter by status, click the *Any Status* arrow and the status you want to filter.
+
+The *Name* column lists the name and the number of connected replicas.
+
+The *Status* column shows the connection status for the replicas associated with a particular gateway.
+
+The following table describes the meaning of each status for a group of replicas:
+
+[%header,cols="30a,70a"]
+|===
+|Status | Group of Replicas
+|Connected | At least one replicas is to the platform and are serving traffic.
```

Comment:
At least one **replica is Connected** to the platform and **is** serving traffic.